pkg/doc/verifiable/example_support_test.go

```diff
@@ -52,64 +47,6 @@ func (udc *UniversityDegreeCredential) MarshalJSON() ([]byte, error) {
 	return json.Marshal(&cp)
 }
 
-func getEd25519Signer(privKey []byte) *ed25519Signer {
-	return &ed25519Signer{privateKey: privKey}
-}
-
-type ed25519Signer struct {
-	privateKey []byte
-}
-
-func (s *ed25519Signer) Sign(doc []byte) ([]byte, error) {
-	if l := len(s.privateKey); l != ed25519.PrivateKeySize {
-		return nil, errors.New("ed25519: bad private key length")
-	}
-
-	return ed25519.Sign(s.privateKey, doc), nil
-}
-
-func getECDSASecp256k1Signer(privKey *ecdsa.PrivateKey) *ecdsaSecp256k1Signer {
-	return &ecdsaSecp256k1Signer{
-		privKey: privKey,
-	}
-}
-
-type ecdsaSecp256k1Signer struct {
-	privKey *ecdsa.PrivateKey
-}
-
-func (es *ecdsaSecp256k1Signer) Sign(payload []byte) ([]byte, error) {
```

Comment:
I see this signing logic removed here and https://github.com/hyperledger/aries-framework-go/pull/1860/files#diff-bc435fee068227dfd806579fb2b37277L222

I believe there were more instances of this code snippet (3 or 4 at least), will they be removed in a later PR?

Reply:
Yes.